model/src/pyrenew/process/deterministic.py

```diff
@@ -0,0 +1,73 @@
+from pyrenew.distutil import validate_discrete_dist_vector
+from pyrenew.metaclass import RandomVariable
+
+
+class DeterministicProcess(RandomVariable):
+    """A wrapper instance of RandomVariable that allows pass deterministic
+    quantities"""
+
+    def __init__(
+        self,
+        vars: tuple,
+        label: str = "a_random_variable",
+        validate_pmf: bool = False,
```

Comment:
I wouldn't declare this construct argument for the abstract base class. Should it be declared only for subclasses designed to represent discrete pmfs?

Indeed, one could have 
```python 
class DiscretePMF(DeterministicProcess)
```

Reply:
This is not intended to be a metaclass. I actually use it in the code. I'll show you som examples later.

Reply:
Even if you envision this as concrete, I'd still suggest making a child class for PMFs.

Reply:
I see your point. Added to the discussion!

Reply:
So perhaps, have a `deterministic` submodule? Sounds like a option, but probably overkill?

Reply:
Added in the latest commit.